Makefile

```diff
@@ -49,6 +49,8 @@ HTTP_PROXY_URL := $(or $(HTTP_PROXY_URL), "")
 HTTPS_PROXY_URL := $(or $(HTTPS_PROXY_URL), "")
 NO_PROXY_VALUES := $(or $(NO_PROXY_VALUES), "")
 VIP_DHCP_ALLOCATION := $(or $(VIP_DHCP_ALLOCATION),yes)
+IPv4 := $(or $(IPv4),yes)
```

Comment:
Why there are two parameters? Could it be like an IP mode? Do we want to support no IPv4?

Reply:
Modes are:
IPv6 only
IPv4 only
Dual

Reply:
1) IPv4, 2) IPv6, 3) IPv4+IPv6. There's a validation that it has to be at lease one.